src/Aquifer.Jobs/local.settings.example.json

```diff
@@ -1,13 +1,13 @@
 {
     "IsEncrypted": false,
     "ConnectionStrings": {
-        "BiblioNexusDb": "Server=localhost;Database=biblionexus-local-db;User Id=sa;Password=Password!;TrustServerCertificate=true;"
+        "BiblioNexusDb": "Server=localhost;Database=biblionexus-local-db;User Id=sa;Password=Password!;TrustServerCertificate=true;",
+        "AzureStorageAccount": ""
```

Comment:
I set up this connection string and two new environment variables below on both the dev and prod Jobs instances.